src/Roslyn.Suppressions/Roslyn.Suppressions/SuppressionChecker.cs

```diff
@@ -117,8 +118,7 @@ public bool IsSuppressed(Diagnostic reportedDiagnostic, string settingsKey)
         }
 
         private static bool IsSameFile(Diagnostic diagnostic, SuppressedIssue suppressedIssue) =>
-            diagnostic.Location.SourceTree?.FilePath.EndsWith(suppressedIssue.FilePath, StringComparison.OrdinalIgnoreCase)
-            ?? false;
+            PathHelper.IsServerFileMatch(diagnostic.Location.SourceTree?.FilePath, suppressedIssue.FilePath);
```

Comment:
Doh! This is the fix for the Roslyn suppression. I meant to exclude it from this PR...